ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -601,8 +602,6 @@ void markCallbackProcessingEnd() {
     public void onCompletion(final BlobInfo routerResult, Exception routerException) {
       callbackTracker.markOperationEnd();
       if (routerResult == null && routerException == null) {
-        frontendMetrics.routerCallbackError.inc();
```

Comment:
I don't think it is useful. This situation is essentially a programming error. The exception is there to help catch it during development itself. Having metrics didn't make much sense to me.